xml/obs_source_service.xml

```diff
@@ -7,154 +7,299 @@
+    <listitem>
+     <para>
+      The serviceonly mode is running the service on the service only. This
+      can be useful, when the service is not available or can not work on
+      developer workstations.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>buildtime</literal> Mode</term>
+    <listitem>
+     <para>
+      The service is running inside of the build job, both for local and
+      server side builds. A side effect is that the service package is
+      becoming a build dependency and must be available. Every user can
+      provide and use a service this way in their projects. The generated
+      sources are not part of the source repository, but part of the
+      generated source packages.
+      Note that services requiring external network access are likely to
+      fail in this mode, because such access is not available if the build
+      workers are running in secure mode<phrase
+       os="opensuse;novell"> (as is always the case at <link
+        xlink:href="https://build.opensuse.org"/>)</phrase>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><literal>disabled</literal> Mode</term>
+    <listitem>
+     <para>
+      The disabled mode is neither running the service locally or on the
+      server side. It can be used to temporarily disable the service but keeping
+      the definition as part of the service definition. Or it can be used to
+      define the way how to generate the sources and doing so by manually calling
+      <command>osc service disabledrun</command> The result will
+      get committed as standard files again.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
  </sect1>
- <sect1>
-  <title>Storage of Source Service Definitions</title>
-  <para> The called services are always defined in a
-    <command>_service</command> file. It is either part of the package sources
-   or used project-wide when stored inside the <command>_project</command>
-   package. </para>
-  <para> The _service file contains a list of services which get called in this
-   order. Each service may define a list of parameters and a mode. The project
-   wide services get called after the per package defined services. The
-   _service file is an XML file like this example:</para>
-   <screen>
-&lt;services&gt;
+
+ <sect1 xml:id="sec.obs.sserv.validate">
+  <title>Defining Services for Validation</title>
+  <para>
+   Source Services may be used to validate sources. This can be definied
+   as:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Per Package</title>
+     <para>
+      Useful when the packager wants to validate the downloaded sources
+      are really from the original maintainer.
+     </para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Per Project</title>
+     <para>
+      Useful for applying general policies which cannot get skipped in
```

Comment:
general->project-wide (?)
get->be
"in" is definitely wrong, but i am not 100% sure of my proposal either: in->for

Reply:
Yes, used project-wide. Fixed.